sql/core/src/test/scala/org/apache/spark/sql/DataFrameFunctionsSuite.scala

```diff
@@ -315,6 +315,49 @@ class DataFrameFunctionsSuite extends QueryTest with SharedSparkSession {
     checkAnswer(df.select(isnotnull(col("a"))), Seq(Row(false)))
   }
 
+  test("Test nullif function") {
+
+    def testNullIfFunction(alwaysInlineCommonExpr: String): Unit = {
+      withSQLConf(SQLConf.ALWAYS_INLINE_COMMON_EXPR.key -> alwaysInlineCommonExpr) {
+
```

Comment:
Could you simplify it a little. Below the common approach in Spark tests too:
```suggestion
    Seq(true, false).foreach { alwaysInlineCommonExpr =>
      withSQLConf(SQLConf.ALWAYS_INLINE_COMMON_EXPR.key -> alwaysInlineCommonExpr.toString) {
```